frontend/pages/hosts/details/DeviceUserPage/DeviceUserPage.tests.tsx

```diff
@@ -148,4 +148,269 @@ describe("Device User Page", () => {
       expect(btn).toBeNull();
     });
   });
+  // // FIXME: revisit these tests when we have a better way to test modals
```

Comment:
Gabe and I spent some time pairing on these tests and concluded we'll need to develop some patterns/tools in order to test modals at this level.